### PR TITLE
Prove +-zero and +-assoc for Conat

### PR DESCRIPTION
**Pull request by freedan42x**

### CODE REVIEW

File: Cubical/Codata/Conat/Properties.agda [hunk 1]
```diff
@@ -83,6 +83,24 @@ n+′∞≡∞′ (suc n) = λ i → suc (n+∞≡∞ n i)
 ∞+∞≡∞ : ∞ + ∞ ≡ ∞
 force (∞+∞≡∞ i) = suc (∞+∞≡∞ i)
 
++-zeroˡ : ∀ n → conat zero + n ≡ n
```
Comment (ice1000) [inline Cubical/Codata/Conat/Properties.agda:86]: I guess you can use `𝟘` for `conat zero`

**Review by ice1000 — APPROVED** [review]
Cool

File: Cubical/Codata/Conat/Properties.agda (hunk 1 above)
Comment (freedan42x) [reply Cubical/Codata/Conat/Properties.agda]: Done!

**Review by ice1000 — APPROVED** [review]
cool

**Comment by mortberg** [comment]
Thank you for the contribution!

**Pull request merged**